core/src/main/java/io/kestra/core/models/executions/Execution.java

```diff
@@ -91,6 +92,19 @@ public class Execution implements DeletedInterface, TenantInterface {
     @Builder.Default
     boolean deleted = false;
 
+    @Builder.Default
+    @With
+    @JsonInclude
```

Comment:
```suggestion
```

Should not be needed

Reply:
The value is not present in the JSON response of the API without it